pages/common/glances.md

```diff
@@ -0,0 +1,23 @@
+# glances
+
+> A cross-platform system monitoring tool.
+
+- run in terminal.
+
+`glances`
+
+- run in web server mode.
+
+`glances -w`
+
+- run in server mode.
```

Comment:
It's unclear what's the difference between this option and the above. Do you think you could clarify it?